Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -8900,16 +8900,16 @@ a_Player:OpenWindow(Window);
 			]],
 			Functions =
 			{
-                                BurnsInDaylight =
```

Comment:
dafaq? IDE? U okaY?

Reply:
It appears it actually corrected a previous change where spaces were used. :)

Reply:
spaces got changed to tabs, I guess it's fine :P